algo/src/main/java/com/hartwig/serve/sources/hartwig/gene/HartwigCuratedGeneFileReader.java

```diff
@@ -0,0 +1,19 @@
+package com.hartwig.serve.sources.hartwig.gene;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.jetbrains.annotations.NotNull;
+
+public final class HartwigCuratedGeneFileReader {
```

Comment:
You could also create a named static factory method to reduce the duplication between HartwigCuratedGeneFileReader and HartwigDriverGeneFileReader and keep the explicit naming.

Reply:
Good one, updated